components/home/home.tsx

```diff
@@ -115,31 +87,7 @@ export const Home: FunctionComponent<HomeProps> = (props) => {
                     />
                 )}
             </Content>
-            {size === "small" && (
-                <Box align="center">
-                    {props.user.profileImages[0] ? (
-                        <Avatar
-                            id="avatar-profile-image-small"
-                            src={props.user.profileImages[0].url}
-                            size={size !== "small" ? "xlarge" : "large"}
-                            border={{ size: "small", side: "all", color: "accent-1" }}
-                            onClick={() => window.open(props.user.profileUrl, "_blank")}
-                            title="click to open your spotify profile"
-                        />
-                    ) : (
-                        <Avatar
-                            id="avatar-default-small"
-                            background="accent-2"
-                            border={{ size: "small", side: "all", color: "accent-1" }}
-                            size={size !== "small" ? "large" : "medium"}
-                            onClick={() => window.open(props.user.profileUrl, "_blank")}
-                            title="click to open your spotify profile"
-                        >
-                            <User color="accent-1" size={size !== "small" ? "large" : "medium"} />
-                        </Avatar>
-                    )}
-                </Box>
-            )}
+            {size === "small" && <UserDetails user={user} small />}
```

Comment:
yeah, i removed this in another branch to free up space, so you can just remove this line entirely and just render the avatar like you're doing on line 55